lab-michael/index.js

```diff
@@ -0,0 +1,5 @@
+var objTest = require('./lib/greeting');
+
+objTest();
```

Comment:
You're not passing in any argument here, so your module is just going to return `hello undefined`.